<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-07-27T19:29:59 IST" name="Zintrl Suite" finished-at="2023-07-27T19:30:47 IST" duration-ms="48679">
    <groups>
    </groups>
    <test started-at="2023-07-27T19:29:59 IST" name="Register at Zintrl" finished-at="2023-07-27T19:30:47 IST" duration-ms="48679">
      <class name="testcases.Register">
        <test-method is-config="true" signature="beforeTest(org.testng.ITestContext)[pri:0, instance:testcases.Register@1c7696c6]" started-at="2023-07-27T19:29:59 IST" name="beforeTest" finished-at="2023-07-27T19:30:00 IST" duration-ms="815" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@510689af]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeMethod(org.testng.ITestContext)[pri:0, instance:testcases.Register@1c7696c6]" started-at="2023-07-27T19:30:00 IST" name="beforeMethod" finished-at="2023-07-27T19:30:00 IST" duration-ms="334" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@510689af]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="create_Account()[pri:0, instance:testcases.Register@1c7696c6]" started-at="2023-07-27T19:30:00 IST" name="create_Account" finished-at="2023-07-27T19:30:45 IST" duration-ms="44828" status="FAIL">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Cannot find elements when the class name expression is null.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Cannot find elements when the class name expression is null.
at org.openqa.selenium.internal.Require$ArgumentChecker.nonNull(Require.java:207)
at org.openqa.selenium.By$ByClassName.<init>(By.java:304)
at org.openqa.selenium.By.className(By.java:99)
at utility.Generic_Keyword.getLocator(Generic_Keyword.java:131)
at utility.Generic_Keyword.presenceOfElement(Generic_Keyword.java:113)
at utility.Generic_Keyword.getElement(Generic_Keyword.java:89)
at utility.Generic_Keyword.click(Generic_Keyword.java:70)
at testcases.Register.create_Account(Register.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- create_Account -->
        <test-method is-config="true" signature="afterTest(org.testng.ITestContext)[pri:0, instance:testcases.Register@1c7696c6]" started-at="2023-07-27T19:30:46 IST" name="afterTest" finished-at="2023-07-27T19:30:47 IST" duration-ms="1274" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@510689af]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method is-config="true" signature="afterSuite(org.testng.ITestContext)[pri:0, instance:testcases.Register@1c7696c6]" started-at="2023-07-27T19:30:47 IST" name="afterSuite" finished-at="2023-07-27T19:30:48 IST" duration-ms="204" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@510689af]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- testcases.Register -->
    </test> <!-- Register at Zintrl -->
  </suite> <!-- Zintrl Suite -->
</testng-results>
